ansible_collections/arista/avd/docs/installation/collection-installation.md

```diff
@@ -77,45 +77,30 @@ collections_paths = ${PWD}/collections:~/.ansible/collections:/usr/share/ansible
 $ ansible-galaxy collection install -U arista.avd
 Process install dependency map
 Starting collection install process
-Installing 'arista.avd:4.4.0' to '/root/.ansible/collections/ansible_collections/arista/avd'
+Installing 'arista.avd:4.8.0' to '/home/arista/.ansible/collections/ansible_collections/arista/avd'
```

Comment:
I can't comment but we can remove the `:` in the note above

```
You can use `-U` to upgrade to a new version for any installed collection: <<<<< this semi column
```

Reply:
I have removed it as a note.